_includes/layouts/base.njk

```diff
@@ -91,6 +91,7 @@
             <a href="{{ entry.url | url }}">{{ entry.title }}</a>{%- endif -%}
           </li>{%- endfor %}
           <li><a href="https://talkingopen.podbean.com/">הסכת - מדברים פתוח</a></li>
+          <li><a href="https://careers-redhat.icims.com/jobs/search?ss=1&searchLocation=13269--Raanana&mobile=false&width=1140&height=500&bga=true&needsRedirect=false&jan1offset=120&jun1offset=180">Careers in Israel</a></li>
```

Comment:
```suggestion
          <li><a href="https://careers-redhat.icims.com/jobs/search?ss=1&searchLocation=13269--Raanana&mobile=false&width=1140&height=500&bga=true&needsRedirect=false&jan1offset=120&jun1offset=180">קריירה בישראל</a></li>
```

Reply:
The rest of the menu is in hebrew, maybe this should be to?
![image](https://user-images.githubusercontent.com/28388442/233863632-a13a9eb5-1379-4dd6-a9a0-727db3a67f3b.png)

Reply:
Thanks. updated.